node/parachain/src/cli.rs

```diff
@@ -58,6 +58,19 @@ pub struct RunCmd {
 	pub parachain_id: Option<u32>,
 }
 
+#[allow(missing_docs)]
```

Comment:
Is this `missing_docs` necessary? It isn't present below on the `pub struct Cli`.

Reply:
Yes, probably the copy/paste betrayed me.